src/utils/DataProcessingUtils.js

```diff
@@ -541,11 +541,41 @@ const replaceEntityAddressKeys = (input :Object | Map, replacer :Replacer) => {
   }, {});
 };
 
+// for each key
+const getEKIDsBySet = (data :Object, entitySetIds :Object) :{ [string] :Set<UUID> } => {
```

Comment:
1. Please spell it out `getEntityKeyIdsBySet`.
2. Please rename this function.

Using `ByThing` implies a collection is indexed / keyed by that thing. Here, it's easy to misinterpret this as a collection indexed by a Set, i.e. `:Map<Set<?>, Set<UUID>>`, which is not the case. I'd recommend `getEntityKeyIdsByEntitySetId`, which is accurate though very verbose. 

The backend has `EntityDataKey`, which is simply the entity set id and the entity key id:

```
public class EntityDataKey {
  private final UUID entitySetId;
  private final UUID entityKeyId;
  ...
}
```

Alternatively, you could name this `getEntityDataKeys`, though even this is not perfect.